### PR TITLE
Simplify command line output

### PR DESCRIPTION
**Pull request by jonamfischer**

### CODE REVIEW

File: htcrystalball/examine.py [hunk 1]
```diff
@@ -166,8 +166,7 @@ def check_slot_by_type(slot: dict, n_cpu: int, ram: float, disk: float,
         slot: The slot to be checked for running the specified job.
         n_cpu: The number of CPU cores for a single job
         ram: The amount of RAM for a single job
-        job_duration: The duration for a single job to execute
-        n_jobs: The number of similar jobs to be executed
+        disk: The number of GPU units for a single job
```
Comment (aqw) [inline htcrystalball/examine.py:169]: Type here. `disk` vs `GPU units`

File: htcrystalball/display.py [hunk 2]
```diff
@@ -67,10 +67,8 @@ def results(result: dict, verbose: bool, matlab: bool,
     # write table and wall-time info to console
     if verbose:
         console.print(table)
-        console.print("USAGE:")
-        console.print("[green]--- [/green]<= 95%")
-        console.print("[yellow]--- [/yellow]> 95%")
-        console.print("[red]--- [/red]> 100%")
+        console.print("LEGEND:")
+        console.print("[green]█[/green] (<= 95%); [yellow]█[/yellow] (95-100%); [red]█[/red] (> 100%)")
```
Comment (jonamfischer) [inline htcrystalball/display.py:71]: Never heard of the "block" glyph before. That's very neat. :-)

Comment (aqw) [reply htcrystalball/display.py:71]: Yep, it's quite useful. It's the one you see at the command prompt, when your cursor is blinking. :-)

**Comment by jonamfischer** [comment]
Ready to merge.

**Pull request merged**